unit_tests/test_data/test_scylla_yaml_builders/longevity-10gb-3h.result.json

```diff
@@ -16,6 +16,6 @@
   ],
   "broadcast_rpc_address": "__NODE_IPV6_ADDRESS__",
   "experimental": true,
-  "prometheus_address": "__NODE_IPV6_ADDRESS__",
-  "enable_ipv6_dns_lookup": true
+  "enable_ipv6_dns_lookup": true,
+  "prometheus_address": "0.0.0.0"
```

Comment:
@enaydanov should be  here the ipv6 all interface address?

Reply:
Good question. I think it should.